frontend/catalyst/api_extensions/error_mitigation.py

```diff
@@ -46,7 +46,9 @@ def mitigate_with_zne(fn=None, *, scale_factors=None, deg=None):
     Args:
         fn (qml.QNode): the circuit to be mitigated.
         scale_factors (array[int]): the range of noise scale factors used.
-        deg (int): the degree of the polymonial used for fitting.
+        extrapolate (Callable): A qjit-compatible function taking two sequences as arguments (scale
+            factors, and results), and returning a float by performing a fitting procedure.
+            By default, perfect polynomial fitting will be used.
```

Comment:
We'll need a description of the `extrapolate_kwargs` as well :)

Reply:
Good catch. Done in https://github.com/PennyLaneAI/catalyst/pull/806/commits/14f02816850b33db6f7bb17640109560c7c74cbc.